pkg/cloudprovider/providers/vsphere/vclib/pbm.go

```diff
@@ -0,0 +1,104 @@
+package vclib
+
+import (
+	"fmt"
+
+	"github.com/golang/glog"
+	"github.com/vmware/govmomi"
+	"github.com/vmware/govmomi/pbm"
+	"github.com/vmware/govmomi/property"
+	"github.com/vmware/govmomi/vim25/mo"
+	"github.com/vmware/govmomi/vim25/types"
+	"golang.org/x/net/context"
+
+	pbmtypes "github.com/vmware/govmomi/pbm/types"
+)
+
```

Comment:
And where GetCompatibleDatastoresMo and GetPlacementCompatibilityResult are called? Are they going to be used later?